elementary/tracking/anonymous_tracking.py

```diff
@@ -109,18 +117,32 @@ def track_cli_end(
             "module_name": module_name,
             "command": command,
         }
+        if self.internal_exceptions:
+            props["internal_exceptions"] = self.internal_exceptions
         self.send_event("cli-end", properties=props)
 
     def track_cli_exception(
         self, module_name: str, exc: Exception, command: str = None
     ) -> None:
         props = {
-            "exception_type": str(type(exc)),
             "module_name": module_name,
             "command": command,
         }
+        props.update(self._get_exception_properties(exc))
+
         self.send_event("cli-exception", properties=props)
 
+    def record_cli_internal_exception(self, exc: Exception):
+        if len(self.internal_exceptions) < self.INTERNAL_EXCEPTIONS_LIMIT:
```

Comment:
Maybe when we pass the limitation we need to add a prop that tells us that?
So we could know at least how many problems there were in the run?

Reply:
Makes sense, I'll add